modules/nodes-nodes-configuring-graceful-shutdown.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: PROCEDURE
+[id="nodes-nodes-activating-graceful-shutdown_{context}"]
+= Configuring graceful node shutdown
+
+To configure graceful node shutdown, create a `KubeletConfig` custom resource (CR) to specify a shutdown grace period for pods on the node. The graceful node shutdown feature minimizes interruption to workloads running on these pods.
+
+[NOTE]
+====
+If you do not configure graceful node shutdown, the default grace period is `0` and the pod is forcefully evicted from the node.
+====
+
+.Prerequisites
+
+* You have access to the cluster with the `cluster-admin` role.
+* You have defined priority classes for pods that require critical or regular classification.
+
+.Procedure
+
+. Define shutdown grace periods in the `KubeletConfig` CR by saving the following YAML in the `kubelet-gns.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: graceful-shutdown
+  namespace: openshift-machine-config-operator
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      pools.operator.machineconfiguration.openshift.io/worker: "" <1>
```

Comment:
Earlier in the document you talk about creating a KubeletConfig to "create a `KubeletConfig` custom resource (CR) to specify a shutdown grace period for pods on the node", but the KubeletConfig actually specifies a machineConfigPoolSelector which selects multiple nodes (in this example, worker nodes).

Reply:
Changed the intro text to `create a `KubeletConfig` custom resource (CR) to specify a shutdown grace period for pods on a set of nodes`. Does this work?